### DISCUSSION TITLE
Auto Setup Postal Script

### DISCUSSION THREAD

**Discussion by CryptosWORLD** [postalserver/postal#1939] [Help with using Postal]
Is Everybody,
is anyone know or can help to build a correct auto Setup Postal Script 
in Debian - Ubuntu and Red Hut - Centos - Rocky Linux etc
?

Thank !

**Comment by uhlhosting** [comment]
I believe the Docker deployment can be called already an autosetup, all you would need is to have docker running.

**Comment by CryptosWORLD** [comment]
NO, i speak for something different
I HAVE TEST the same command lines from the official website to setup POSTAL and in different VPS I HAD different RESULTS
I mean .. with the same commands in the same Linux OS full updated i try to :
to setup docker + composer + postal
1 setup goes well and the other 3 not!
 
I m looking for a script that gives a complete setup ...with results of a correct Postal Setup.

**Reply by willpower232** [reply]
Postal requires an instance of RabbitMQ and MySQL/MariaDB but doesn't want to be responsible for managing them so I'd imagine your problems were related to those.

Also those dependencies are slower to start than Postal so its possible that your problems required a bit more waiting time to be resolved.

**Comment by CryptosWORLD** [comment]
I think many issues will be stopped if all use the same Postal setup script.

**Reply by uhlhosting** [reply]
@CryptosWORLD do not forget this is still an open source project, everyone is free to PR and add value. In my knowledge there should be some ansible deployments, not sure how up to date they are.
